curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3c866d28d7ad0de6470505.md

```diff
@@ -16,19 +16,22 @@ Add the class name `flavor` to the `French Vanilla` `p` element.
 You should add the `flavor` class to your `p` element.
 
 ```js
-assert(code.match(/<p\s*class\s*=\s*('|")flavor\1\s*>/i));
+const el = document.querySelector(".flavor");
+assert.exists(el);
 ```
 
 You should only have one element with the `flavor` class.
 
 ```js
-assert($('.flavor').length === 1);
+const elements = document.querySelectorAll('.flavor');
+assert.lengthOf(elements, 1);
 ```
 
 Your `flavor` class should be on the `p` element with the text `French Vanilla`.
 
 ```js
-assert($('.flavor')[0].innerText.match(/French Vanilla/i));
+const el = document.querySelector(".flavor");
+assert.match(el.innerText,/French Vanilla/i);
```

Comment:
Shouldn't this be an equal check for the string value and not a case-insensitive regex match?

Reply:
I don't think it really matters that much here. As long as the text is close enough, it's probably fine.

Reply:
For the sake of clarity, I am going to apply this suggestion.